src/frontend/helpers/gamepad.ts

```diff
@@ -300,13 +300,37 @@ export const initGamepad = () => {
     }
   }
 
+  function isValveGamepad(gamepad: Gamepad | null) {
+    return gamepad && gamepad.id.includes('Vendor: 28de')
+  }
+
+  function filterValveGamepads(gamepads: (Gamepad | null)[]) {
+    return gamepads.filter(isValveGamepad)
+  }
+
+  function isMaskedGamepad(
+    valveGamepads: (Gamepad | null)[],
+    gamepad: Gamepad
+  ) {
+    return valveGamepads.find(
+      (valveGamepad) =>
+        valveGamepad &&
+        Math.abs(valveGamepad.timestamp - gamepad.timestamp) <= 10
```

Comment:
why is this needed btw?

Reply:
sadly the timestamps of the masked gamepad and steam input gamepad are not identical. most of the time there is a difference of 5, but sometimes it is up to 8. Therefore i used 10 to be sure to catch the masked gamepad. sadly the nearly identical timestamp is the only thing they have in common.